src/control/server/ctl_storage_rpc.go

```diff
@@ -755,6 +772,10 @@ func (cs *ControlService) StorageScan(ctx context.Context, req *ctlpb.StorageSca
 			State: new(ctlpb.ResponseState),
 		}
 	} else {
+		if err := cs.cleanSpdkResources(); err != nil {
+			return nil, errors.Wrap(err, "clean spdk resources")
+		}
+
```

Comment:
@NiuYawei, do you know if this is safe? I think for StorageFormat it's okay, but my knowledge is insufficient in the StorageScan case.